components/crm/.env.example

```
@@ -15,6 +15,7 @@ MONGO_PASSWORD=lerian
 MONGO_PORT=5703
 MONGO_MAX_POOL_SIZE=1000
 MONGO_PARAMETERS=
+MONGO_TLS_CA_CERT=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MONGO_TLS_CA_CERT key should go before the MONGO_URI key

(UnorderedKey)